suites/pacific/rados/tier-2_rados_basic_regression.yaml

```diff
@@ -347,6 +347,12 @@ tests:
           sleep_interval: 60
       desc: Ceph balancer plugins CLI validation in upmap mode
 
+  - test:
+      name: ceph health alerts test
+      polarion-id: CEPH-83573240
```

Comment:
This Test case speaks about verification of alerts on dashboard. The code generates alerts and verifies the same in the ceph status in the cephadm shell. Is this the correct test ID?

Reply:
Yes dashboard alerts are coming from ceph health status hence I added that ID,
If you know any other tests in polarion let me know we can mark them also automated
also can we use the same polarion ID for other alerts to be automated